packages/build/src/utils/blobs.ts

```diff
@@ -12,6 +14,40 @@ export const getBlobsDirs = (buildDir: string, packagePath?: string) => [
   path.resolve(buildDir, packagePath || '', LEGACY_BLOBS_PATH),
 ]
 
+interface EnvironmentContext {
+  api?: {
+    host: string
+    scheme: string
+  }
+  deployId?: string
+  siteId?: string
+  token?: string
+}
+
+// TODO: Move this work to a method exported by `@netlify/blobs`.
+export const getBlobsEnvironmentContext = ({
+  api = { host: DEFAULT_API_HOST, scheme: 'https' },
+  deployId,
+  siteId,
+  token,
+}: EnvironmentContext) => {
+  if (!deployId || !siteId || !token) {
+    return {}
+  }
+
+  const payload = {
```

Comment:
Could we perhaps add some sort of note that this doesn't include all potential values (i.e. edgeUrls, region), just the ones we needed at the time of adding this? Or better yet, can we go all the way and expose the function in `@netlify/blobs` like we are planning to do anyhow?